test/generator/expected_output/compiled_output.coffee

```diff
@@ -12,13 +12,13 @@ class CompiledOutput
             var _v2 = ( ( 5 * 6 ) - _v1 );
             var _v3 = 5.5;
             var _v4 = ( ! true );
-            var _v5 = (function(lb, ub, skip) {
-                var temp = [];
```

Comment:
@rtoal HATES `temp` - I suspect even in compilers. Hopefully `buffer` is a descriptive enough variable name.